src/index.html

```diff
@@ -5,9 +5,168 @@
+        </div>
+      </div>
+      <h1 class="header__title">Miami<br>
+        Condo Kings</h1>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--active">Home</li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">About us</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">gallery</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">consultation</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">testimonials</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    </header>
+    <main>
+      <section class="section">
+        <article class="info">
+          <h2 class="info__title">Miami best <br> Real estate
+            <a class="info__link" href="#">here</a>
+          </h2>
+          <div class="info__text">
+            <p class="info__text--active">Over 50 years experience in love.</p>
+          <p>Mazel and VA Tsukkerman in his "Analysis of musical works."<br> Pointillism, which originated in the music of the early twentieth <br> century, microforms, found a distant historical.<br></p>
+          <br>
+          <p>The market leader in real estate since 2003. Awards Real Estate <br> Company, 2010, 2011, 2013. More than 200 satisfied customers<br> premium segment. We focus only on new buildings and high-level<br> doskanalno know this area better than other specialists. <br>Working directly with developers - so have more useful information about the profitable deals.<br></p>
+          <br>
+          <p><strong>Will never be late! <br> Guaranteed secure transaction - 100%</strong></p>
+          </div>
+        </article>
+      </section>
+      <section class="section">
+        <h2 class="section__heading heading">Gallery
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="gallery">
+          <div class="photos photos-wide-left">
+            <img class="photos__img" src="images/photos/1.jpg" alt="maymi-view1">
+            <img class="photos__img" src="images/photos/2.jpg" alt="maymi-view2">
+          </div>
+          <p class="gallery__caption">
+            <a class="photos__link" href="#">Marina Palms</a> / North Miami Beach, FL 33162</p>
+          <div class="photos">
+            <img class="photos__img" src="images/photos/3.jpg" alt="maymi-view3">
+            <img class="photos__img" src="images/photos/4.jpg" alt="maymi-view4">
+          </div>
+          <p class="gallery__caption">
+            <a class="photos__link" href="#">Edition Residens</a> / Miami Beach, FL 33139</p>
+          <div class="photos photos-wide-right">
+            <img class="photos__img" src="images/photos/5.jpg" alt="maymi-view5">
```

Comment:
```suggestion
            <img class="photos__img" src="images/photos/5.jpg" alt="maymi-view5">
```
делай atl более осмысленным

Reply:
done